internal/consumer/dlqMetadata.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+message dlqMetadata {
+    int64 retry_count = 1;
+    string topic = 2;
+    int32 partition = 3;
+    int64 offset = 4;
+    int64 timestamp_ns = 5;
```

Comment:
Go naming style: timestamp_nanos
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
I prefer timestamp_ns and naming conventions allows for an exception for protobuf generated code.